qualtran/bloqs/arithmetic/conversions.py

```diff
@@ -65,7 +65,7 @@ def signature(self) -> Signature:
         )
 
     def on_classical_vals(
-        self, mu: 'ClassicalValT', nu: 'ClassicalValT'
+        self, *, mu: 'ClassicalValT', nu: 'ClassicalValT'
```

Comment:
why are these all key-word only now?

Reply:
Mainly I was having fun with my pylint check.

The system will always call this method with keyword arguments, and users probably shouldn't be calling it directly. The real issue is if you accept position only arguments (including `*args`) it will break